t/obs/SUSE:ALP:Products:Bedrock:0.1/print_openqa.before

```diff
@@ -76,13 +76,13 @@
 
 /usr/bin/openqa-cli api -X post isos?async=1 \
  ARCH=x86_64 \
- ASSET_1=ALP-Bedrock.x86_64-0.1-Default-encrypted-Build2.2.raw \
  ASSET_256=ALP-Bedrock.x86_64-0.1-Default-encrypted-Build2.2.raw.sha256 \
  BUILD=2.2 \
- CHECKSUM_ASSET_1=$(cut -b-64 /var/lib/openqa/factory/other/ALP-Bedrock.x86_64-0.1-Default-encrypted-Build2.2.raw.sha256 | grep -E '[0-9a-f]{5,40}' | head -n1) \
+ CHECKSUM_HDD_1=$(cut -b-64 /var/lib/openqa/factory/other/ALP-Bedrock.x86_64-0.1-Default-encrypted-Build2.2.raw.sha256 | grep -E '[0-9a-f]{5,40}' | head -n1) \
```

Comment:
This is the important change to review, the other files are just build number changes.